Riot/Modules/Room/TimelineDecorations/Reactions/RoomReactionActionViewCell.swift

```diff
@@ -54,6 +68,18 @@ final class RoomReactionActionViewCell: UICollectionViewCell, NibReusable, Thema
         self.updateViews()
     }
     
+    func fill(actionIcon: UIImage) {
+        let attachment = NSTextAttachment()
+        attachment.image = actionIcon.vc_resized(with: CGSize(width: self.actionLabel.bounds.size.height, height: self.actionLabel.bounds.size.height))?.withRenderingMode(.alwaysTemplate)
+
+        let attachmentString = NSAttributedString(attachment: attachment)
```

Comment:
Not sure to understand why you don't use directly `let attachmentString = NSAttributedString(attachment: attachment)` otherwise do not hesitate to add a comment if there is a formatting issue

Reply:
You're definitely right :)